src/CMakeLists.txt

```diff
@@ -293,10 +293,14 @@ TARGET_LINK_LIBRARIES (FORTE_LITE ${LINK_LIBRARY})
 
 list(JOIN link_flags " " link_flags_string)
 
+if("${FORTE_ARCHITECTURE}" STREQUAL "Posix")
+  set(FORTE_LINK_STATIC_LIBC_LIBSTDCPP OFF CACHE BOOL "Force FORTE linking statically against libc and lib stdc++")
```

Comment:
Since this enables the `-static` flag which affects all libraries, just call this `FORTE_LINK_STATIC`

Reply:
Agreed.